examples/social-media/backend/asyncapi.yaml

```diff
@@ -19,64 +19,65 @@ servers:
       mqtt:
         clientId: websocketServer
 channels:
-  comment/liked:
+  notifyAllCommentLiked:
     address: comment/liked
     messages:
       subscribe.message:
-        $ref: ../common/messages.yaml#/commentLiked
+        $ref: '../common/messages.yaml#/commentLiked'
     description: Notify all the services that a comment has been liked.
     servers:
       - $ref: '#/servers/mosquitto'
-  like/comment:
+  newLikeComment:
     address: like/comment
     messages:
       publish.message:
-        $ref: ../common/messages.yaml#/likeComment
+        $ref: '../common/messages.yaml#/likeComment'
     description: When a comment like is received from the frontend.
     servers:
       - $ref: '#/servers/websiteWebSocketServer'
-  'comment/{commentId}/changed':
+  commentsCountChange:
     address: 'comment/{commentId}/changed'
     messages:
       publish.message:
-        $ref: ../common/messages.yaml#/commentChanged
+        $ref: '../common/messages.yaml#/commentChanged'
     description: >-
       When an event from the broker arrives telling us to update the comment
       likes count on the frontend.
     servers:
       - $ref: '#/servers/mosquitto'
     parameters:
-      commentId: {}
-  update/comment/likes:
+      commentId:
+        $ref: '../common/parameters.yaml#/commentId'
```

Comment:
since it is no longer schema it is better to promote also a separate file per parameters